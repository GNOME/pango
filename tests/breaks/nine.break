# indic
मी काच खाऊ शकतो, मला ते दुखत नाह
मैं काँच खा सकता हूँ, मुझे उस से कोई पीडा नहीं होत
நான் கண்ணாடி சாப்பிடுவேன், அதனால் எனக்கு ஒரு கேடும் வராத
ﻢﯾں ﮎﺎﻨﭼ ﮎھﺍ ﺲﮑﺗﺍ ہﻭں ﺍﻭﺭ ﻢﺟھے ﺖﮑﻠﯿﻓ ﻥہﯼں ہﻮﺘﯾ 
ﺰﻫ ﺶﻴﺸﻫ ﺥﻭړﻝې ﺶﻣ، ﻪﻐﻫ ﻡﺍ ﻦﻫ ﺥﻭږﻮﻳ
