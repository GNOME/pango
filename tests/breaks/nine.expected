Text:         ⁦म⁩ ⁦ी⁩  [ ]  ⁦क⁩ ⁦ा⁩ ⁦च⁩  [ ]  ⁦ख⁩ ⁦ा⁩ ⁦ऊ⁩  [ ]  ⁦श⁩ ⁦क⁩ ⁦त⁩ ⁦ो⁩  ⁦,⁩ [ ]  ⁦म⁩ ⁦ल⁩ ⁦ा⁩  [ ]  ⁦त⁩ ⁦े⁩  [ ]  ⁦द⁩ ⁦ु⁩ ⁦ख⁩ ⁦त⁩  [ ]  ⁦न⁩ ⁦ा⁩ ⁦ह⁩  [0x0a]  ⁦म⁩ ⁦ै⁩ ⁦ं⁩  [ ]  ⁦क⁩ ⁦ा⁩ ⁦ँ⁩ ⁦च⁩  [ ]  ⁦ख⁩ ⁦ा⁩  [ ]  ⁦स⁩ ⁦क⁩ ⁦त⁩ ⁦ा⁩  [ ]  ⁦ह⁩ ⁦ू⁩ ⁦ँ⁩  ⁦,⁩ [ ]  ⁦म⁩ ⁦ु⁩ ⁦झ⁩ ⁦े⁩  [ ]  ⁦उ⁩ ⁦स⁩  [ ]  ⁦स⁩ ⁦े⁩  [ ]  ⁦क⁩ ⁦ो⁩ ⁦ई⁩  [ ]  ⁦प⁩ ⁦ी⁩ ⁦ड⁩ ⁦ा⁩  [ ]  ⁦न⁩ ⁦ह⁩ ⁦ी⁩ ⁦ं⁩  [ ]  ⁦ह⁩ ⁦ो⁩ ⁦त⁩  [0x0a]  ⁦ந⁩ ⁦ா⁩ ⁦ன⁩ ⁦்⁩  [ ]  ⁦க⁩ ⁦ண⁩ ⁦்⁩ ⁦ண⁩ ⁦ா⁩ ⁦ட⁩ ⁦ி⁩  [ ]  ⁦ச⁩ ⁦ா⁩ ⁦ப⁩ ⁦்⁩ ⁦ப⁩ ⁦ி⁩ ⁦ட⁩ ⁦ு⁩ ⁦வ⁩ ⁦ே⁩ ⁦ன⁩ ⁦்⁩  ⁦,⁩ [ ]  ⁦அ⁩ ⁦த⁩ ⁦ன⁩ ⁦ா⁩ ⁦ல⁩ ⁦்⁩  [ ]  ⁦எ⁩ ⁦ன⁩ ⁦க⁩ ⁦்⁩ ⁦க⁩ ⁦ு⁩  [ ]  ⁦ஒ⁩ ⁦ர⁩ ⁦ு⁩  [ ]  ⁦க⁩ ⁦ே⁩ ⁦ட⁩ ⁦ு⁩ ⁦ம⁩ ⁦்⁩  [ ]  ⁦வ⁩ ⁦ர⁩ ⁦ா⁩ ⁦த⁩  [0x0a]  ⁦ﻢ⁩ ⁦ﯾ⁩ ⁦ں⁩  [ ]  ⁦ﮎ⁩ ⁦ﺎ⁩ ⁦ﻨ⁩ ⁦ﭼ⁩  [ ]  ⁦ﮎ⁩ ⁦ھ⁩ ⁦ﺍ⁩  [ ]  ⁦ﺲ⁩ ⁦ﮑ⁩ ⁦ﺗ⁩ ⁦ﺍ⁩  [ ]  ⁦ہ⁩ ⁦ﻭ⁩ ⁦ں⁩  [ ]  ⁦ﺍ⁩ ⁦ﻭ⁩ ⁦ﺭ⁩  [ ]  ⁦ﻢ⁩ ⁦ﺟ⁩ ⁦ھ⁩ ⁦ے⁩  [ ]  ⁦ﺖ⁩ ⁦ﮑ⁩ ⁦ﻠ⁩ ⁦ﯿ⁩ ⁦ﻓ⁩  [ ]  ⁦ﻥ⁩ ⁦ہ⁩ ⁦ﯼ⁩ ⁦ں⁩  [ ]  ⁦ہ⁩ ⁦ﻮ⁩ ⁦ﺘ⁩ ⁦ﯾ⁩  [ ] [0x0a]  ⁦ﺰ⁩ ⁦ﻫ⁩  [ ]  ⁦ﺶ⁩ ⁦ﻴ⁩ ⁦ﺸ⁩ ⁦ﻫ⁩  [ ]  ⁦ﺥ⁩ ⁦ﻭ⁩ ⁦ړ⁩ ⁦ﻝ⁩ ⁦ې⁩  [ ]  ⁦ﺶ⁩ ⁦ﻣ⁩  ⁦،⁩ [ ]  ⁦ﻪ⁩ ⁦ﻐ⁩ ⁦ﻫ⁩  [ ]  ⁦ﻡ⁩ ⁦ﺍ⁩  [ ]  ⁦ﻦ⁩ ⁦ﻫ⁩  [ ]  ⁦ﺥ⁩ ⁦ﻭ⁩ ⁦ږ⁩ ⁦ﻮ⁩ ⁦ﻳ⁩  [0x0a]  
Breaks:     c    c    lc   c c    lc   c c    lc c c   c  c   lc c   c    lc   c    lc   c c c    lc   c c       Lc     c    lc     c c    lc   c    lc c c   c    lc     c  c   lc   c   c    lc c c    lc   c    lc   c c    lc   c   c    lc c     c    lc   c c       Lc   c   c    lc c   c   c   c    lc   c   c   c   c   c   c  c   lc c c   c   c    lc c c   c   c    lc c   c    lc   c   c   c    lc c   c c       Lc c c c    lc c c c c    lc c c c    lc c c c c    lc c c c    lc c c c    lc c c c c    lc c c c c c    lc c c c c    lc c c c c    c      Lc c c    lc c c c c    lc c c c c c    lc c c  c   lc c c c    lc c c    lc c c    lc c c c c c       Lc
Whitespace:      x           x           x                x          x         x             x           w              x             x         x             x              x            x         x         x           x             x             x           w                x                   x                                x                x                 x           x                 x             w              x             x           x             x           x           x             x               x             x             x    w           x             x               x            x          x         x         x               w       w 
Sentences:  bs                                                                                           e       bs                                                                                                                                               e       bs                                                                                                                                           e       bs                                                                                                                               e           bs                                                                                             e       b 
Words:      bs   be   bs     be   bs     be   bs       be b   bs     be   bs   be   bs       be   bs     be      bs     be   bs       be   bs   be   bs       be   bs     be b   bs       be   bs   be   bs   be   bs     be   bs       be   bs       be   bs     be      bs       be   bs             be   bs                       be b   bs           be   bs           be   bs     be   bs           be   bs       be      bs     be   bs       be   bs     be   bs       be   bs     be   bs     be   bs       be   bs         be   bs       be   bs       be   b      bs   be   bs       be   bs         be   bs   be b   bs     be   bs   be   bs   be   bs         be      b 
Graphemes:  b    b    b    b b    b    b b    b  b b   b  b   b  b   b    b    b    b    b b b    b    b b       b      b    b      b b    b    b    b  b b   b    b      b  b   b    b   b    b  b b    b    b    b    b b    b    b   b    b  b     b    b    b b       b    b   b    b  b   b   b   b    b    b   b   b   b   b   b  b   b  b b   b   b    b  b b   b   b    b  b   b    b    b   b   b    b  b   b b       b  b b b    b  b b b b    b  b b b    b  b b b b    b  b b b    b  b b b    b  b b b b    b  b b b b b    b  b b b b    b  b b b b    b      b  b b    b  b b b b    b  b b b b b    b  b b  b   b  b b b    b  b b    b  b b    b  b b b b b       b 
Hyphens:       i         i i         i i         i i i           i i         i         i i i         i i            i i         i i i         i         i i i         i i           i i i         i         i         i i         i i i         i i i         i i            i i i         i i i i i i         i i i i i i i i i i i           i i i i i         i i i i i         i i         i i i i i         i i i            i i         i i i         i i         i i i         i i         i i         i i i         i i i i         i i i         i i i                i         i i i         i i i i         i           i i         i         i         i i i i           
